tests/e2e/Makefile

```diff
@@ -31,7 +31,7 @@ e2e-test:
 		echo "[ERR] KUBECONFIG missing, must be defined"; \
 		exit 1; \
 	fi
-	$(GO_CMD) test -v $(CURDIR)/tests/e2e -args \
+	cd $(CURDIR)/tests/e2e && $(GO_CMD) test -v . -args \
```

Comment:
Question: Do we need to cd back to the project root afterwards?

Reply:
no, that's handled by Make, I had the same question my self, but after a manual run I saw my self on the same PATH